content/announcements/2022-02-24-update-ross.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Ross Nicoll Stepping into an Advisor Role for the Dogecoin Foundation"
+date = "2022-02-24"
+[ author ]
+  name = "Dogecoin Foundation"
++++
+
+
+Recently Ross [made public his decision](https://rnicoll.name/posts/2022-02-16-stepping-away/) to 
+step back from involvement in the Foundation as a Director, and more generally from 
+active development in the crypto-space. 
```

Comment:
```suggestion
active development in the crypto space. 
```

Reply:
(https://github.com/dogecoinfoundation/foundation.dogecoin.com/blob/377b0a4ed426e3b5e9bd17c9c56d211525d8a298/content/advisories/2021-08-31-advisory-dogecoin2.md#L12) is written as 2 words.